src/doc/footer.inc

```diff
@@ -5,5 +5,5 @@ or the <a href="http://opensource.org/licenses/MIT">MIT license</a>, at your opt
 </p><p>
 This file may not be copied, modified, or distributed except according to those terms.
 </p></footer>
-<script type="text/javascript" src="jquery.js"></script>
-<script type="text/javascript" src="playpen.js"></script>
+<script type="text/javascript" src="/jquery.js"></script>
+<script type="text/javascript" src="/playpen.js"></script>
```

Comment:
Is there any documentation on contributing to docs specifically? Is there a constraint that all links in docs should run when browsing on the local file-system? For example, it's pretty easy for a contributor to just run `python -m SimpleHTTPServer` in the docs directory and have absolute links work correctly.